apps/api/src/application/controller/WebSocketController.ts

```diff
@@ -0,0 +1,26 @@
+import type { WSContext, WSEvents } from 'hono/ws';
+
+export const ws_stocks = new Map();
+
+export function WebSocketHandler(stock_id: string): WSEvents {
+  if (!ws_stocks.has(stock_id)) {
+    ws_stocks.set(stock_id, new Set<WSContext>());
+  }
+  const ws_stock = ws_stocks.get(stock_id);
+
+  const ws: WSEvents = {
+    onOpen(event, socket) {
```

Comment:
```suggestion
    onOpen(_event, socket) {
```
使用しない変数は先頭にアンダーバーを入れておいてください！！